build.gradle

```diff
@@ -14,6 +14,7 @@ buildscript {
     ext.androidxCoreVersion = "1.8.0"
     ext.mockwebserverVersion = "4.2.0"
     ext.tinkVersion = "1.8.0"
+    ext.coroutinesVersion = "1.7.1"
```

Comment:
This coroutines version is incompatible with current Kotlin version https://github.com/RevenueCat/purchases-android/blob/7f157c7dafd7af12f8f28b34b2a89d579fc1fbd1/build.gradle#L2

https://app.circleci.com/pipelines/github/RevenueCat/purchases-android/5918/workflows/f9d92590-c2ed-41c8-b4b0-6568fedf9a76/jobs/17498

```
> Task :purchases:kaptGenerateStubsDefaultsDebugUnitTestKotlin FAILED
e: /home/circleci/.gradle/caches/transforms-3/42eea213d57537de4e15f013b44a2e42/transformed/jetified-kotlin-stdlib-common-1.8.20.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/circleci/.gradle/caches/transforms-3/6da424c6e0c554c67d3ef8fa5ab80860/transformed/jetified-kotlinx-coroutines-core-jvm-1.7.1.jar!/META-INF/kotlinx-coroutines-core.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/circleci/.gradle/caches/transforms-3/7af5082d687f4dcda2f059f53fae807c/transformed/jetified-kotlin-stdlib-1.8.20.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/circleci/.gradle/caches/transforms-3/7af5082d687f4dcda2f059f53fae807c/transformed/jetified-kotlin-stdlib-1.8.20.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/circleci/.gradle/caches/transforms-3/7af5082d687f4dcda2f059f53fae807c/transformed/jetified-kotlin-stdlib-1.8.20.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /home/circleci/.gradle/caches/transforms-3/ac99cb870f3b12e83c96ccf1bea39152/transformed/jetified-kotlinx-coroutines-test-jvm-1.7.1.jar!/META-INF/kotlinx-coroutines-test.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
```

Latest compatible with Kotlin `1.6.21` is https://github.com/RevenueCat/purchases-android/blob/df1b8a4c764a60542853431d3483b4846a3d17e0/build.gradle#L17

Is it ok to bump the Kotlin version to the latest though?

Reply:
We don't want to bump to Kotlin 1.8 yet until we have done more research. On our tests, it looks like apps using 1.6 break if we update. So I will update this to 1.6.4, which is compatible with Kotlin 1.6.21

Reply:
Makes sense 👍